packages/klighd-core/src/di.config.ts

```diff
@@ -68,6 +68,7 @@ import { EDGE_TYPE, LABEL_TYPE, NODE_TYPE, PORT_TYPE, SKEdge, SKLabel, SKNode, S
 import { SetSynthesesAction, SetSynthesisAction } from './syntheses/actions'
 import { SynthesesRegistry } from './syntheses/syntheses-registry'
 import { KEdgeView, KLabelView, KNodeView, KPortView, SKGraphView } from './views'
+import { searchBarModule } from './search-bar/searchbar-module'
```

Comment:
Please sort the imports according to the alphabet in two ways: imports of a single package should be alphabetically ordered (not relevant here, as there is only one) and all packages should also be alphabetically ordered (./search-bar before ./sidebar)